### PR TITLE
fixes sample script loading issue

### PR DESCRIPTION
**Pull request by esotrope**
image

### CODE REVIEW

File: webapp/src/main/webapp/src/app/admin/ingest-pipeline/model/pipeline.ts [hunk 1]
```diff
@@ -89,21 +89,6 @@ export interface PipelineScript extends Script {
...
-
-  /**
-   * The script publish time
-   */
-  publishedOn?: Date;
-
-  /**
-   * The user who published the script
-   */
-  publishedBy?: string;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/admin/ingest-pipeline/model/pipeline.ts]: unrelated cleanup

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineScriptService.java [hunk 2]
```diff
@@ -38,7 +35,7 @@ public PipelineScript get(final int pipelineId, final int id) {
 
         // if there is a saved script return it
         final PipelineScript savedScript = repository.findById(id);
-        if (savedScript != null) {
+        if (savedScript != null && savedScript.getPipelineId() == pipelineId) {
```
Comment (esotrope) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineScriptService.java]: unrelated "security"

File: webapp/src/main/webapp/src/app/admin/ingest-pipeline/page/pipeline/pipeline.component.ts [hunk 3]
```diff
@@ -166,7 +176,10 @@ export class PipelineComponent implements ComponentCanDeactivate, OnDestroy {
         ]) => {
           const pipeline = {
             ...basePipeline,
-            script,
+            script: script || {
+              id: -1,
+              pipelineId: basePipeline.id
+            },
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/admin/ingest-pipeline/page/pipeline/pipeline.component.ts]: wonky but gets the job done by putting a placeholder script in that results in a sample script lookup

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineScriptService.java (hunk 2 above)
Comment (burzum619) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineScriptService.java]: why no `===`?

Comment (esotrope) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultPipelineScriptService.java]: ha! if only java let me - i would do it to troll people
